### ISSUE TITLE
Support for more Music Apps (e.g Spotify, YT Music, etc)

### ISSUE THREAD

**Issue by tdr1995** [ClutchEngineering/Sidecar#89]
Just thought it would be nice (as a Spotify user myself) if it supported more apps for the Now Playing widget, if possible!

**Comment by Jomino1** [comment]
Agreed! Support for Audible would also be nice, or video thumbnails for YouTube videos if watching/listening to them in the background.

**Comment by cachatj** [comment]
i often am using YouTube or Doppler for audio. Doppler is a barebones music app to get streaming middle men outta your music library (itunes, spotify wtf these days) remember the good ol days when ../mp3/ is all ya needed & it worked no matter what?!

**Comment by cachatj** [comment]
sorry got on a rant there. 

point of comment was to say, that Sidecar & other displays in vehicle have a hard time picking this up correctly

**Comment by jverkoey** [comment]
Got a functional demo of Spotify Now Playing working :D

<img width="459" alt="Image" src="https://github.com/user-attachments/assets/f1d1ca97-4ff8-49b5-b212-a5e89516cd48" />

This is still a bit far from the finish line of getting approved for Spotify developer access, but it's one step closer!
